content/concepts/stream-multiplexing.md

```diff
@@ -2,17 +2,21 @@
 title: Stream Multiplexing
 ---
 
-Stream Multiplexing (often abbreviated as "stream muxing") allows multiple independent logical streams to all share a common underlying transport medium.
+Stream Multiplexing (_stream muxing_) is a way of sending multiple streams of data over one communication link. It combines multiple signals into one unified signal so it can be transported 'over the wires', then it is demulitiplexed (_demuxed_) so it can be output and used by separate applications.
+
+//what type of multiplexing? Time-division? Frequency division because blocks?
```

Comment:
Is this a leftover comment?

Reply:
Yes, wasn't actually ready to merge, want to touch on tomorrow @mxinden